tekton/triggerbinding.yml

```diff
@@ -0,0 +1,45 @@
+---
+apiVersion: tekton.dev/v1alpha1
+kind: TriggerBinding
+metadata:
+  name: lightwalletd-master-binding
+spec:
+  params:
+  - name: newSHA
+    value: $(body.after)
+---
+apiVersion: tekton.dev/v1alpha1
+kind: TriggerTemplate
+metadata:
+  name: lightwalletd-dockerbuild-template
+spec:
+  params:
+  - name: newSHA
+    description: The git repository HEAD sha   
+  resourcetemplates:
```

Comment:
I'm curious why the `TaskRun` resource template defined here is identical to the `TaskRun` resource template in the `taskruns.yml` file above

Reply:
Because that way you can run the task independently from the trigger. It was really just part of the setup and test process, but I could see it being useful on it's own.